.pipelines/e2e.yml

```diff
@@ -21,6 +21,7 @@ jobs:
     parameters:
       azureDevOpsJSONSPN: $(aro-v4-e2e-devops-spn)
   - script: |
+      az account set -s 46626fc5-476d-41ad-8c76-2ec49c6994eb
```

Comment:
should this use a variable instead of hardcoding the sub id here?

Reply:
oh nvm I see your comment in the description :sweat_smile:

Reply:
I couldn't see the variables and now I can.  Updated it, we'll see if it passes e2e.